sky/packages/sky/lib/rendering/box.dart

```diff
@@ -1452,19 +1466,12 @@ class RenderImage extends RenderBox {
   void paint(PaintingContext context, Offset offset) {
     if (_image == null)
       return;
-    bool needsScale = size.width != _image.width || size.height != _image.height;
-    final PaintingCanvas canvas = context.canvas;
-    if (needsScale) {
-      double widthScale = size.width / _image.width;
-      double heightScale = size.height / _image.height;
-      canvas.save();
-      canvas.translate(offset.dx, offset.dy);
-      canvas.scale(widthScale, heightScale);
-      offset = Offset.zero;
-    }
-    canvas.drawImage(_image, offset.toPoint(), _paint);
-    if (needsScale)
-      canvas.restore();
+    paintImage(canvas: context.canvas,
```

Comment:
for consistency, I would indent this the same way we indent constructors with many arguments.